### PR TITLE
Added sdk for Stream Sharing

### PR DESCRIPTION
**Pull request by stkr89**
This PR adds sdk for Stream Sharing

### CODE REVIEW

File: cdx/v1/api/openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,6304 @@
...
+  x-api-group: cdx/v1
+  x-tag-group: Data Exchange API (%s)
+  x-audience: company-internal
+servers:
+- description: Confluent Cloud production
+  url: https://api.confluent.cloud
+- description: Confluent Cloud staging
+  url: https://api.stag.cpdev.cloud
+- description: Confluent Cloud development
+  url: https://api.devel.cpdev.cloud
```
Comment (brianstrauch) [inline cdx/v1/api/openapi.yaml:21]:
```suggestion
```

**Review by brianstrauch — APPROVED** [review]

**Review by linouk23 — CHANGES_REQUESTED** [review]
We might want to add it to `confluentinc/ccloud-sdk-go-v2` first.

**Comment by stkr89** [comment]
> We might want to add it to `confluentinc/ccloud-sdk-go-v2-internal` first.

@linouk23 It's already added there. [https://github.com/confluentinc/ccloud-sdk-go-v2-internal/tree/master/cdx](https://github.com/confluentinc/ccloud-sdk-go-v2-internal/tree/master/cdx)

**Review by linouk23 — APPROVED** [review]
✅

**Pull request merged**